server/engine/src/main/java/org/eclipse/lsp/cobol/core/preprocessor/delegates/copybooks/GrammarPreprocessorListenerImpl.java

```diff
@@ -186,31 +169,4 @@ public void enterEveryRule(ParserRuleContext ctx) {
     ThreadInterruptionUtil.checkThreadInterrupted();
     super.enterEveryRule(ctx);
   }
-
-  @Override
-  public void enterReplaceAreaStartOrOffStatement(ReplaceAreaStartOrOffStatementContext ctx) {
```

Comment:
Copybook processing now do not handle `REPLACE` clause and is handled later on in the chain